userbot/modules/waifu.py

```diff
@@ -0,0 +1,52 @@
+# Imported from ppe-remix
+
+import re
+import random
+from userbot import CMD_HELP, bot
+from userbot.events import register
+
+
+EMOJI_PATTERN = re.compile(
+    "["
+    "\U0001F1E0-\U0001F1FF"  # flags (iOS)
+    "\U0001F300-\U0001F5FF"  # symbols & pictographs
+    "\U0001F600-\U0001F64F"  # emoticons
+    "\U0001F680-\U0001F6FF"  # transport & map symbols
+    "\U0001F700-\U0001F77F"  # alchemical symbols
+    "\U0001F780-\U0001F7FF"  # Geometric Shapes Extended
+    "\U0001F800-\U0001F8FF"  # Supplemental Arrows-C
+    "\U0001F900-\U0001F9FF"  # Supplemental Symbols and Pictographs
+    "\U0001FA00-\U0001FA6F"  # Chess Symbols
+    "\U0001FA70-\U0001FAFF"  # Symbols and Pictographs Extended-A
+    "\U00002702-\U000027B0"  # Dingbats
+    "]+")
+
+
+def deEmojify(inputString: str) -> str:
+    return re.sub(EMOJI_PATTERN, '', inputString)
+
+
+@register(outgoing=True, pattern="^.waifu(?: |$)(.*)")
+async def waifu(animu):
+    text = animu.pattern_match.group(1)
+    if not text:
+        if animu.is_reply:
+            text = (await animu.get_reply_message()).message
+        else:
+            await animu.answer("`No text given, hence the waifu ran away.`")
+            return
+    animus = [20, 32, 33, 40, 41, 42, 58]
+    sticcers = await bot.inline_query(
+        "stickerizerbot", f"#{random.choice(animus)}{(deEmojify(text))}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/MoveAngel/One4uBot/pullRequest?prid=5573699)